plan/cbo_test.go

```diff
@@ -300,6 +300,12 @@ func (s *testAnalyzeSuite) TestAnalyze(c *C) {
 
 	testKit.MustExec("create table t3 (a int, b int)")
 	testKit.MustExec("create index a on t3 (a)")
+
+	testKit.MustExec("create table t4 (a int, b int, c int)")
+	testKit.MustExec("create index a on t4 (a)")
+	testKit.MustExec("create index b on t4 (b)")
+	testKit.MustExec("insert into t4 (a,b,c) values (1,1,1),(1,2,1),(1,3,1),(1,4,1),(2,5,2),(2,6,2),(2,7,2),(2,8,2)")
```

Comment:
Don't need to insert data and don't need a new table, just add one more index to t3 and test on it.